environment.yml

```
@@ -10,29 +10,37 @@
-      - bandit==1.7.9
-      - safety==3.2.3
+      - pytest-asyncio==0.21.0
+      - pytest-cov==6.2.1
+      - ruff==0.6.9
+      - black==23.7.0
       - mypy==1.10.0
-      - python-dotenv==1.0.1
-      - psycopg2-binary==2.9.9
+      - bandit==1.7.9
+      - pre-commit==3.3.0
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)